pod.go

```diff
@@ -547,11 +547,6 @@ func createPod(podConfig PodConfig) (*Pod, error) {
 		return nil, err
 	}
 
-	// Passthrough devices
-	if err := p.attachDevices(); err != nil {
```

Comment:
Please also remove this function implementation from `pod.go` since nothing else actually uses it.